ts/services/NewReportsService.ts

```diff
@@ -0,0 +1,75 @@
+import * as moment from 'moment';
+import * as caporal from 'caporal';
+import * as request from 'request-promise';
+
+class NewReportsService {
```

Comment:
Is this because there's already a reports service?

Reply:
yea, I decided to go this way. New reports use completely new APIs so why not we split them out? This way it looks better structured to me